### DISCUSSION TITLE
Failed to pull image "docker.io/splunk/splunk-operator:2.3

### DISCUSSION THREAD

**Discussion by Zeroxcharisma** [splunk/splunk-operator#1189] [Polls]
Warning  Failed     73s (x5 over 2m33s)  kubelet            Error: ImagePullBackOff
  Normal   Pulling    59s (x4 over 2m35s)  kubelet            Pulling image "docker.io/splunk/splunk-operator:2.3"
  Warning  Failed     58s (x4 over 2m34s)  kubelet            Failed to pull image "docker.io/splunk/splunk-operator:2.3": rpc error: code = NotFound desc = failed to pull and unpack image "docker.io/splunk/splunk-operator:2.3": failed to resolve reference "docker.io/splunk/splunk-operator:2.3": docker.io/splunk/splunk-operator:2.3: not found

the problem is he is trying to pull a wrong image , i dunno why he is not taking .0 at the last of the image as it is mentioned in the chart .

**Comment by vivekr-splunk** [comment]
we are working on fixing this chart issue should be done soon.

**Comment by akondur** [comment]
The issue is resolved in latest releases. Please take a look & re-open this thread if the problem still exists.

**Comment by Zeroxcharisma** [comment]
the issues doesn't exist anymore , thank you for checking this with me
again
